### ISSUE TITLE
🐞 Bug Report: Equal height not working in loop builder (ED-8948)

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#20645]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hi elementor team.

It seems that Equal height does not work properly in the loop builder. Please see the following video :

https://user-images.githubusercontent.com/77163248/206143854-0111fa6f-fdcc-480d-8340-ff8181f6cea1.mp4

### Steps to reproduce

It is enough to add the loop grid and activate this option in case of uneven height. It will not work properly.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

[system-info-localhost-07-12-2022.txt](https://github.com/elementor/elementor/files/10175105/system-info-localhost-07-12-2022.txt)

**Comment by 79ho3ein** [comment]
Strangely, this option does not work for me under any circumstances. Either in the section or in the container

**Comment by 79ho3ein** [comment]
Hi @nicholaszein 
I apologize for the delay. Yes I did. I really checked every way I could think of to identify the problem, but unfortunately I still haven't found the problem. Could it be related to the php version?

**Comment by JakJakson** [comment]
+1  same issue here

1 section, 1 column

**Comment by 79ho3ein** [comment]
Hi again @nicholaszein , I just tested this bug on one of my live websites and it worked correctly. I don't know why, but it seems to be a problem with localhost.
But I tested [this bug](https://github.com/elementor/elementor/issues/20347) on the live website and it still occurs...

**Comment by nicholaszein** [comment]
Thanks for the feedback @79ho3ein! 🙏 

I'll mark this issue as solved then. Thanks for letting me know it worked correctly!

About that issue, I'll soon reach there to answer you! 😉

Thanks for being so patient! 💪

**Comment by nicholaszein** [comment]
Hello @79ho3ein and @JakJakson!

We have great news! 🙌

We were able to identify this happening with sections and columns, and a fix is going to be introduced with v3.10!

Thanks for being so patient! 🙏

**Comment by paraimpacto** [comment]
@nicholaszein I have Elementor v3.10 running, and the issue persists. Please see the screencast:

https://disk.yandex.com/i/f4RmwW40ue3F0w

**Comment by 79ho3ein** [comment]
Hi @paraimpacto 

Everything works fine for me. Maybe it's related to other plugins.

elementor : 3.10.0
elementor pro : 3.10.1
Hello Elementor : 2.6.1

**Comment by paraimpacto** [comment]
Did a plugin conflict. No dice. Suggestions? It was only updated 3days ago on GitHub.

**Comment by 79ho3ein** [comment]
Please do the following steps:

1- Make sure that Elementor, Elementor Pro and hello Elementor are updated.
2- Disable all plugins except Elementor and Elementor Pro.
3- Change your theme to hello elementor.

As a final test, in the Elementor settings > tools, regenerate files and data.

If the problem persists, add your system information here for the Elementor team to investigate.

**Comment by paraimpacto** [comment]
@79ho3ein I. believe that I already explained that I. did a plugin conflict, but for the sake of the doubt, please find attached a a screenshot. This is a loop built with elementor loop builder pulled into an elementor look grid with only elementor and elementor pro with the hello elementor as the theme, all other plugins disabled:

![IMG_6852](https://user-images.githubusercontent.com/33914588/212786119-0b4f9ccc-5999-47fc-ae20-a72f1939aa58.png)

**Comment by paraimpacto** [comment]
Here is the system info:

[system-info-x2z8cxp7uw-staging.onrocket.site-17-01-2023 (1).txt](https://github.com/elementor/elementor/files/10430176/system-info-x2z8cxp7uw-staging.onrocket.site-17-01-2023.1.txt)

**Comment by perks98** [comment]
@paraimpacto on your loop item template where have you applied the styling for the border and white background? Did you apply on the parent section or the column?

**Comment by paraimpacto** [comment]
@perks98 tried both, no changes either way. Intuitively, the equal height toggle would just apply the adjustment to the loop grid, (like many other page builders do). I am not quite sure what the issue could be with this.

**Comment by paraimpacto** [comment]
Update: Elementor Tier 2 support has recognized this as a bug related to using the loop and loop grid with sections and columns. Although it seems to be functional with flexbox and containers (for those who are confident that those elenets are stable), there will be a fix in the changelog at some point for sections and columns (as it has already been announced as compatible). 

 https://docs.elementor.com/article/178-changelog​

**Comment by niallpattersonmedia** [comment]
This seems to be an issue for the loop carousel now, works fine in the grid but not carousel, see attached video 
 pro current versions and only plugins active.
 
![image](https://user-images.githubusercontent.com/79975755/221161875-d79c7626-01cb-4281-8863-8f18c84145be.png)


**Video Link**
https://www.dropbox.com/s/m0dmebzyeuf0jjk/Screen%20Recording%202023-02-24%20at%2010.51.05.mov?dl=0

**Comment by ph-webetc** [comment]
Equal height loop grid is working on old sites with sections; it's working on sites built from scratch with containers.
However, I can't get it to work on sites where sections have been converted to containers...

**Comment by paraimpacto** [comment]
Will there be no chance of it being able to work on sites that were converted to containers in the future? You guys are really pushing the move to containers, and I was just about to go forward, but after reading this now I don’t want to lose the option for equal height. Particularly as we start to go into flex box, equal height is key. Is there any projected fix for this?

**Comment by jagdish1o1** [comment]
I was facing the same issue, but moving the loop element into the container fix the issue for me.

**Comment by heikkiwaan** [comment]
Still this bug is present.
I converted my sections to containers, and can't get the equal height working.

**Comment by 79ho3ein** [comment]
@heikkiwaan 

Activate the optimize DOM option in the Elementor settings.

**Comment by aurabyte** [comment]
This is still a bug for me. I am using containers and Flexbox, with optimize DOM enabled. The equal heights option is not working at all. When will this be fixed? Is there a custom code workaround we can use in the meantime?

**Comment by pintolog** [comment]
I'm having this bug also,
Elementor Pro 3.13.2 
Elementor Theme updated
Did anyone solved this?
![image](https://github.com/elementor/elementor/assets/7497319/74ee3ca0-0c65-47b8-b7ab-f8151ba0bb97)

**Comment by kpc-tanaka** [comment]
No... No matter what I do, it doesn't happen yesterday. I am terribly discouraged. I wonder why the officials won't fix it. Does anyone use Flexbox and know how to make it work?

**Comment by perks98** [comment]
Works for me. @kpc-tanaka If you can link the page you have the loop grid on with the issue I can have a quick look to see if the structure of your loop items is right. Can't guarantee I can help. With containers in loops try and make sure there is only one parent container, and the same with sections. See screenshot of what I mean:

![Screenshot 2023-08-01 093714](https://github.com/elementor/elementor/assets/55829113/0f5d6640-3aef-4256-87f3-a8217d988108)

**Comment by kpc-tanaka** [comment]
@perks98 
I don't think it's a problem.
https://gifu-fair.info/hidagyu_syutoken/
At first glance it may seem like they are the same height, but they are not.
<img width="324" alt="image" src="https://github.com/elementor/elementor/assets/123909655/b2e3249e-be7e-4d9b-a67c-578fd35f1b15">

**Comment by nicholaszein** [comment]
Hi everyone, this issue has already been long fixed. If you're still experiencing any problems with setting up equal heights, it is very likely you are missing something, like for example nesting containers.

Make sure to review your designs.

If you have found a bug, using ONLY the latest version of Elementor, Elementor Pro, and the Hello Elementor theme, and you can consistently reproduce it. Submit a new Bug Report Issue. This issue has been resolved and is not going to be reopened.

Kind regards
